controllers/con-news.js

```diff
@@ -75,3 +75,12 @@ exports.getUsers = (req, res, next) => {
     })
 }
 
+exports.deleteCommentByID = (req, res, next) => {
+    const comment_id = req.params.comment_id
+    removeCommentById(comment_id).then((comment) => {
+        res.status(204).send()
```

Comment:
Excellent, we have nothing to send so it's correct to leave .send() empty - well done!